batch/batch/front_end/front_end.py

```diff
@@ -1327,7 +1336,8 @@ async def ui_get_billing_projects(request, userdata):
     db = request.app['db']
     billing_projects = await _query_billing_projects(db)
     page_context = {
-        'billing_projects': billing_projects
+        'billing_projects': [dict(p, size=len(p['users'])) for p in billing_projects if not p['closed']],
```

Comment:
What does the `size=` do?

Reply:
The size is purely for page formatting reasons. I adjusted the billing project display page so that the project name is a multi-row cell that spans all of the user rows (so `rowspan=len(users) + 1`); jinja didn't like when I tried to compute this within the template.